### ISSUE TITLE
Assign, On hold, and Completed should be read as tabs with AT

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12137]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role

The elements Assign, On Hold and completed are read as tab button windows with assistive technology programs. Functionally these elements are tabs and should be read as tabs. (New: 08/21/2019)

- Probably needs to be fixed in the higher level tab component
